### PR TITLE
Added Page 3 of the Signup process

### LINKED ISSUE #36: Create Page 3 of the Signup process
Anxiety questions 2

This is the design for the third page of the signup process. Create a new page identical to the above picture. No functionality is needed.

Use the `chips_choice_null_safety` [library](https://pub.dev/packages/chips_choice_null_safety) to build the chips.

The `progress bar`, which is located at the top, is excluded from the page.
The `Next` button, which is located at the bottom, is excluded from the page.

Use `SingleChildScrollView` as a parent widget to avoid overflow.

Notes:
- Update your local project before working on the issue
- File names should use the `CamelCase` convention
- Variable names should use the `lowerCamelCase` convention
- Frequently used components should be placed inside the `lib/components` folder
- Pages should be placed inside the `lib/pages` folder
- Format your code before creating a PR
- Remove any unused imports

### PR DESCRIPTION
**Pull request by chandansgowda**
Closes #36 

I have setup the required UI as show in the attached screenshot. I couldn't find a way to change the size of the choicechips. As far as I know, there is no option to change the size of the choicechips in the package we have used. Please correct me if I am wrong.

ag

### CODE REVIEW

File: lib/pages/sign_up_page3.dart [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+              ' such as therapy or medication?',
+              style: GoogleFonts.poppins(fontSize: 20),
+            ),
+            ChipsChoice<int>.single(
+              value: _ans2,
+              onChanged: (value) => setState(() {
+                _ans2 = value;
+              }),
+              direction: Axis.vertical,
+              crossAxisAlignment: CrossAxisAlignment.center,
```
Comment (markhallak) [inline lib/pages/sign_up_page3.dart:86]: Change the direction to horizontal. Also, add direction and crossAxisAlignment to the first ChipsChoice.

The second ChipsChoice should appear similar to the first ChipsChoice. In other words, the chips should be centered and displayed horizontally.

**Comment by markhallak** [comment]
Don't include the `main.dart` file in the PR.

**Comment by chandansgowda** [comment]
Hey @markhallak 
Since the main.dart file had flutter sample code, I have cleaned it. That's why I have committed the file. This will not be done from the next time.

**Comment by markhallak** [comment]
Thanks for that but it is not required for this PR. I will be updating the `main.dart` file very soon.

Please remove it from the commit so I can merge this PR into the branch. Thanks!

**Comment by chandansgowda** [comment]
Hey @markhallak 
I have come up with this situation for the first time. I ended up deleting the main.dart in my PR now. Can you please help me with what has to be done now?
Shall I create a new flutter project and copy paste the sample main.dart file ?

**Comment by markhallak** [comment]
You deleted the file from your local project or removed it from the commit?

Also, did you change the direction in the second ChipsChoice?

File: lib/pages/sign_up_page3.dart (hunk 1 above)
Comment (markhallak) [reply lib/pages/sign_up_page3.dart:86]: Please make the changes mentioned here. Thanks!

**Comment by chandansgowda** [comment]
I removed it from the commit. What change of direction? I didn't get you :/

File: lib/pages/sign_up_page3.dart (hunk 1 above)
Comment (markhallak) [reply lib/pages/sign_up_page3.dart:86]: @chandansgowda check here

**Comment by markhallak** [comment]
I have mentioned you in the review

**Comment by markhallak** [comment]
Did you see the comments in the review?

**Comment by chandansgowda** [comment]
Which review ??

**Comment by markhallak** [comment]
I think if you clicked to the "files changed" tab and scrolled down a bit, you will be able to see my comments on the code.

**Comment by markhallak** [comment]
Screenshot_20230306_232454_Brave

**Comment by chandansgowda** [comment]
Hey @markhallak 
Done with the changes requested. Please review.

**Review by markhallak — APPROVED** [review]

**Pull request merged**